opentelemetry-sdk/tests/instrumentation/test_configurator.py

```diff
@@ -87,10 +87,10 @@ class TestTraceInit(TestCase):
     def setUp(self):
         super()
         self.get_provider_patcher = patch(
-            "opentelemetry.distro.TracerProvider", Provider
+            "opentelemetry.sdk.instrumentation.TracerProvider", Provider
         )
         self.get_processor_patcher = patch(
-            "opentelemetry.distro.BatchSpanProcessor", Processor
+            "opentelemetry.sdk.instrumentation.BatchSpanProcessor", Processor
```

Comment:
These tests only really check these helpful `Configurator` methods so I thought they made sense to move completely to the `opentelemetry-sdk` package. There's nothing "`opentelemetry-distro`" related about them really...